doc/version_history.rst

```diff
@@ -6,10 +6,17 @@
 Version History
 ===============
 
+v1.21.0
```

Comment:
We have not released 1.20.0 yet, so you can make these new scripts part of that release instead.

Reply:
Updated to v1.20.1